File: python/7 kyu/246.py
# https://www.codewars.com/kata/56bd9e4b0d0b64eaf5000819/train/python

# Passed

def combine(*kwargs):
    result = dict()
    for dictionary in kwargs:
        for key in dictionary:
            if key not in result:
                result[key] = 0
                
            result[key] += dictionary[key]
            
    return result

output = combine({ 'a': 10, 'b': 20, 'c': 30 }, { 'a': 3, 'c': 6, 'd': 3 })
print(output)

File: nodejs/7 kyu/246.js
// https://www.codewars.com/kata/59b11f57f322e5da45000254/train/javascript

// Passed

function onesComplement(n) {
  let complement = n
    .split("")
    .map((ch) => (ch === "1" ? "0" : "1"))
    .join("");
  return complement;
}

const output = onesComplement("1101");
console.log(output);